_posts/2018-07-13-FR-team.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Programming Historian French initiative 
+authors:
+- Anna-Maria Sichani
+layout: post
+categories: posts
+---
+
+<p><figure><img src="/avatars/Marie-Puren.png" alt=""/><figcaption>
+<figure><img src="/avatars/Sofia-Papastamkou.png" alt=""/><figcaption>
+Marie Puren and Sofia Papastamkou have joined the Programming Historian</figcaption></figure>
+</p>
+
+We are pleased to announce that Marie Puren and Sofia Papastamkou have joined the Programming Historian project team. By welcoming Marie and Sofia as members of the editorial board we are also introducing the French-speaking initiative of the Programming Historian. 
+
+Marie Puren Ph.D., is a junior researcher in Digital Humanities at the French National Institute for computer science and applied mathematics (Inria) in Paris, member of the ALMAnaCH team (INRIA – Ecole Pratique des Hautes Etudes). As collaborator to the [PARTHENOS] (http://www.parthenos-project.eu/) H2020 project, she focuses her research on the development of standards for data management and research tools in Arts and Humanities. Marie also contributes to the [IPERION] (http://www.iperionch.eu/) H2020 project.
```

Comment:
Do not put a space in between the `[]` part of a markdown URL and the `()` part. See https://programminghistorian.org/en/lessons/getting-started-with-markdown#links

Reply:
done!